src/main/java/edu/ie3/io/factory/EntityData.java

```diff
@@ -34,6 +39,22 @@ public EntityData(
     return fieldsToAttributes;
   }
 
+  public boolean containsKey(String key) {
+    return fieldsToAttributes.containsKey(key);
+  }
+
+  public String get(String field) {
+    return fieldsToAttributes.get(field);
+  }
+
+  public UUID getUUID(String field) {
+    return UUID.fromString(get(field));
+  }
+
+  public <Q extends Quantity<Q>> ComparableQuantity<Q> get(String field, Unit<Q> unit) {
```

Comment:
No as I see the implementation I think I know why you sticked to `get`. Please addy Quantity anyway and javadocs for clarification. Nice work to move this here though! 👍 

Proposal: might be optional or try/catch for safety needed. Otherwise if the value cannot be found and parsed, this might explode?!

Reply:
Yes, makes sense. I implemented exception handling for all get*()-methods.